x-pack/plugins/security_solution/server/endpoint/services/metadata/endpoint_metadata_service.ts

```diff
@@ -431,6 +431,7 @@ export class EndpointMetadataService {
         // the agent object
         const agent: typeof _agent = {
           ..._agent,
+          policy_revision: _agent.policy_revision ?? _agent.policy_revision_idx ?? 0,
```

Comment:
Whats the root cause of the problem? Why is `policy_revision` not in the agent record?

(note: don't hold this PR up based on my question.This change looks like a work-around, so I'm wondering what the real root cause is here. and - is `policy_revision` and `policy_revision_idx` the same thing?